### ISSUE TITLE
don't fail if phantomjs is not present

### ISSUE THREAD

**Issue by damian0604** [uvacw/inca#449]
The activation of the scrapers with wrong filenames (as per PR #448 ) made INCA break on systems where phantomJS is not present. make sure that this is not a hard requirement

**Comment by laurasav** [comment]
Yes, especially because PhantomJS is deprecated, also scrapers for dynamic websites should switch to another solution. Currently the .get()-methods of "corp_kpn_scraper.py" "corp_randstad_scraper.py" (in inca/inca/scrapers) make use of PhantomJS.
